ios/FluentUI/People Picker/AvatarView.swift

```diff
@@ -545,6 +619,32 @@ open class AvatarView: UIView {
         }
     }
 
+    private func updateImageViewWithFallbackImage(style: AvatarFallbackImageStyle = .primaryFilled) {
+        let imageName = style.imageName(size: avatarSize)
+        if let image = UIImage.staticImageNamed(imageName) {
+            let containerSize = avatarSize.size
+            let renderer = UIGraphicsImageRenderer(size: containerSize)
+
+            let personImage = renderer.image { _ in
+                let personImageSize = avatarSize.personImageSize
+                image.draw(at: CGPoint(x: floor((containerSize.width - personImageSize) / 2), y: floor((containerSize.height - personImageSize) / 2)))
+            }.withRenderingMode(.alwaysTemplate)
+            setupWithImage(personImage)
+
+            if let window = window {
+                imageView.backgroundColor = style.backgroundColor(for: window)
+            }
```

Comment:
Are we handle the view did move to window callback to update the image?

Reply:
if I understand your question correctly..... I think my changes in layoutSubviews() handles your scenario?